data/processors/export.py

```diff
@@ -87,7 +87,7 @@ def export_for_api(data, path):
         if "roomfinder_data" in export_data[_id]:
             del export_data[_id]["roomfinder_data"]
         if "props" in export_data[_id]:
-            to_delete = list(filter(lambda e: e != "computed", export_data[_id]["props"].keys()))
+            to_delete = list(filter(lambda e: e not in {"computed", "links"}, export_data[_id]["props"].keys()))
```

Comment:
I always find these filter expressions really hard to parse. What do you think?
```suggestion
            to_delete = [e for e in export_data[_id]["props"].keys() if e not in {"computed", "links"}]
```